content/projects/gsoc/2019/plugin-installation-manager-tool-cli.adoc

```diff
@@ -26,25 +26,74 @@ There are currently many reincarnations of plugin management across Jenkins, inc
+
+=== Phase 2
+Phase 2 focused on improving upon what was written in Phase 1, with particular focus on making parsing more robust,
+adding better Docker compatibility, and creating new CLI options that will make the plugin installation process
+more transparent. Parsing was improved to filter out comments and blank lines, and accept a yaml file listing plugins to install
+ as input. Instead of hard coding the update centers, options were added to allow update centers to be set via environment variables or CLI options.
+Windows compatible defaults for locations to find an existing Jenkins war file and setting the plugin directory were also added.
+To make the plugin installation process more transparent, options were added to view the list of plugins that would be installed prior
+to installation, and view if any of these plugins had available updates or security warnings.  This last step involved re-writing
+some of the plugin download and dependency resolution logic so that all plugins and their recursive dependencies are resolved
+before the time of download.
+
+The blog post announcing the new features can be found link:/blog/2019/07/30/plugin-management-tool-phase2-updates/[here] and the demo
+for this coding phase can be found link:https://www.youtube.com/watch?v=HlENuZZq7zc&feature=youtu.be&t=221[here]
+(link:https://docs.google.com/presentation/d/1wsM09IGgoA7gSsA9tW5fHlFAhl6B9fXlr-4iEP89SCk/edit?usp=sharing[slides])
+
+=== Phase 3
+Most of this phase was dedicated to cleaning up and testing the work done in Phase 1 and 2. Some fixes and optimizations included in
+this coding phase include throwing an exception sooner if there are problems downloading a plugin or resolving its dependencies,
+throwing an exception if the required Jenkins version listed for any of the plugins to be installed is lower than the version
+of the Jenkins war file, and improving plugin download times through a custom IO thread pool and using http HEAD instead of http GET where appropriate.
+
+I was also able to attend DevOps World and present the work I've done on this project both during the Continuous Delivery
+Contributor Summit and for the link:https://devopsworldjenkinsworld2019.sched.com/event/SQhG/community-lunch-time-demos[Jenkins Community Demos].
+
+image::/images/post-images/gsoc-plugin-management-tool/devopsworld.png[DevOps World Demo]
+
+I detailed my time at DevOps World link:/blog/2019/08/2019-08-22-devops-world.adoc[here].
+
 
 === Project Information
 * You can find more information about the original the project proposal link:https://docs.google.com/document/d/1lMCDqY5TKVXyFl67BmyMkaS9GTjRbueKr7ds395b_10/edit?usp=sharing[here].
 * Code can be found link:https://github.com/jenkinsci/plugin-installation-manager-tool[here]
 * Jira tasks and feature requests can be seen link:https://issues.jenkins-ci.org/browse/JENKINS-58199?jql=project%20%3D%20JENKINS%20AND%20component%20%3D%20plugin-installation-manager-tool[here]
```

Comment:
Wouldn't it be better to highlight all the Jira task related to this plugin by using this link 
https://issues.jenkins-ci.org/issues/?jql=project%20%3D%20JENKINS%20AND%20labels%20%3D%20plugin-manager

Reply:
Sure, I can change that. Is there a preference for using labels over components?